ledger/controllers/registrar.js

```diff
@@ -161,7 +161,7 @@ const createPersona = function (runtime, apiVersion) {
         octets: Joi.string().optional().description('octet string that was signed and digested')
       }).required()
     }
-    var validity = (Joi.validate(request.payload, requestSchema).error)
+    var validity = Joi.validate(request.payload, requestSchema)
```

Comment:
nice find! do we want to validate `request.payload.request` instead of `request.payload`?

with this fix CI is failing with `"message": "child \\"headers\\" fails because [\\"headers\\" is required]"` - I see we use `request.payload.request.headers` below so I was thinking we might just be validating at the wrong level. thanks!